src/Core.UnitTests/Notifications/NotificationServiceTests.cs

```diff
@@ -75,6 +76,7 @@ public void ShowNotification_ShouldAddInfoBarOnUiThread()
             runOnUiAction.Should().NotBeNull();
             runOnUiAction();
 
+            VerifySubscribedToInfoBarEvents(infoBar);
```

Comment:
This is what was failing the unit tests after the upgrade: when you call `infoBarManager.VerifyNoOtherCalls`, it was stupidly complaining that setups on infoBar were not verified. I assume this is because the infoBar has events and these invocations are verified somehow differently (seeing as nothing else broke). So the fix was simple, we already had the helper methods to verify the event invocations.